jupyterlab/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qsharp_jupyterlab",
-  "version": "0.0.0",
+  "version": "0.0.7",
```

Comment:
This version is copied in a bunch of places. Do we have a strategy or script for keeping them in sync when we update versions?

Reply:
I can develop that (would be another good step for me in getting more experience with Python).

(I was also thinking about refactoring the version in ".ado\publish.yml" such that the version there is also a single string literal rather than 3 separate vars. Such that the script could just find the same literal, e.g. "0.0.7" in specific lines in particular files (Cargo.toml, package.json, maybe ReadMe.md) and replace just that and nothing unrelated)

Reply:
The ver update script is done. To be pushed.

Reply:
[./update_ver.py](https://github.com/microsoft/qsharp/pull/466/files#diff-6fa024341c08c6c1e358e6c4e046c99edeeddd0f5de7446dd65c18b45847b4cc)